src/detect-http-server-body.c

```diff
@@ -91,7 +91,7 @@ void DetectHttpServerBodyRegister(void)
     sigmatch_table[DETECT_HTTP_RESPONSE_BODY].flags |= SIGMATCH_NOOPT;
     sigmatch_table[DETECT_HTTP_RESPONSE_BODY].flags |= SIGMATCH_INFO_STICKY_BUFFER;
 
-    g_file_data_buffer_id = DetectBufferTypeRegister("file_data");
+    g_http_server_body_buffer_id = DetectBufferTypeRegister("http_server_body");
```

Comment:
this registration as 'file_data' is intentional. The file_data buffer inspect http server body in HTTP, file data in SMTP attachments, etc. The http_server_body modifier implements file_data for http.

Reply:
Will revert these changes.